{"links": {"self": {"href": "data/repositories/osrf/cloudsimk/pullrequests/26/comments/1612234.json"}, "code": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsimk/diff/osrf/cloudsimk:d27768188d47..b1a970ad21ab?path=app%2Fcontrollers%2Fsimulations.js"}, "html": {"href": "#!/osrf/cloudsimk/pull-requests/26/_/diff#comment-1612234"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 26, "links": {"self": {"href": "data/repositories/osrf/cloudsimk/pullrequests/26.json"}, "html": {"href": "#!/osrf/cloudsimk/pull-requests/26"}}, "title": "Select all simulations checkbox"}, "content": {"raw": "Does this check for `err` guarantee that the user was not found? Is it possible for a `err` to be defined, and also have the `sim_id` updated?", "markup": "markdown", "html": "<p>Does this check for <code>err</code> guarantee that the user was not found? Is it possible for a <code>err</code> to be defined, and also have the <code>sim_id</code> updated?</p>", "type": "rendered"}, "created_on": "2014-04-08T20:31:57.940135+00:00", "user": {"display_name": "Nate Koenig", "uuid": "{c862cdd9-fcc8-4419-9fc7-e20db14b8fcb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D"}, "html": {"href": "https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:095b1e12-74ed-4e20-b44f-2f0745b616e0/ca24bb11-4787-4b14-a20d-d91835e9bde2/128"}}, "nickname": "Nathan Koenig", "type": "user", "account_id": "557058:095b1e12-74ed-4e20-b44f-2f0745b616e0"}, "inline": {"to": null, "from": 101, "outdated": true, "path": "app/controllers/simulations.js"}, "updated_on": "2014-04-08T20:31:57.948870+00:00", "type": "pullrequest_comment", "id": 1612234}